### DISCUSSION TITLE
error: [WinError 1920] The file cannot be accessed by the system: 'C:\\Users\\user\\AppData\\Local\\Microsoft\\WindowsApps\\python3.11.exe\\python311.dll'

### DISCUSSION THREAD

**Discussion by subjoel** [marcelotduarte/cx_Freeze#1916] [Q&A]
hi, whenever i try to make an exe from my python script, i run python setup.py build and it gives me this output:

running build
running build_exe
error: [WinError 1920] The file cannot be accessed by the system: 'C:\\Users\\user\\AppData\\Local\\Microsoft\\WindowsApps\\python3.11.exe\\python311.dll'

anyone know how to fix?

**Comment by marcelotduarte** [comment]
Are you using Python from Windows Store?
Use the Python from python.org
